tasks/system_probe.py

```diff
@@ -62,6 +62,7 @@ def build(
     bundle_ebpf=False,
     parallel_build=True,
     kernel_release=None,
+    debug=False,
```

Comment:
I think it'd be helpful to have an example comment here like 
`Example usage: inv -e system-probe.build --debug --kernel-release 5.14.0-1024-oem`. This is where I'd go to look for how to use system-probe.build, and it's not immediately obvious that `DEBUG=1 inv -e system-probe.build` will successfully build WITHOUT DEBUG (this format was suggested by Will which makes me think that this format by be used successfully in other parts of our workflow). `inv -e system-probe.build --debug=True` fails, which a user could reasonably (imho) try because the default is defined as `False`. and It's not immediately obvious to me where `kernel-release` becomes `kernel_release`.